### ISSUE TITLE
Unexpected Error Report

### ISSUE THREAD

**Issue by sawao** [avocode/avocode#618]
IMPORTANT: Please try to describe how did it happen.

Error occurred in the sync with sketch ver.3.5 :scream:
-----
Sketch Exporter 2.1.4 running in Sketch 3.5
```
Error: Error: Command failed: 2016-01-27 16:48:04.640 sketchtool[568:51955] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer)
2016-01-27 16:48:04.641 sketchtool[568:51955] Error: caught exception whilst decoding (version 76):*** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer)
Failed to open document file:///Users/uxpersona/Dropbox/creww-sawa/UI%20Design/Monthly%20Sucscription/Tutorial/Tutorial.sketch
((null) MSDocumentDomain:11)


    at /Users/uxpersona/…/avocode-exporter/lib/image-exporter.js:1:873
    at ChildProcess.exithandler (child_process.js:651:7)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:753:16)
    at Socket.<anonymous> (child_process.js:966:11)
    at Socket.EventEmitter.emit (events.js:95:17)
    at Pipe.close (net.js:465:12)
```

**Comment by bolekkerous** [comment]
Hi,
Sketch 3.5 was released just few hours ago and broke our plugin unfortunately. We're working on a fix that should be ready later today.

If you need to use Avocode before that download Sketch 3.4.4 for now: http://download.sketchapp.com/sketch-3.4.4.zip

If there is a problem with sketch file versions, you can either use the revert-to option in Sketch or check this artilcle about hidden folder with previous Sketch revisions:
https://medium.com/sketch-app-sources/how-to-recover-50-go-or-even-more-by-deleting-sketch-caches-files-e5829dba20e1#.ktu3bcz81

You can follow the status in the issue #609, I am closing this issue as duplicate. Thanks for your report.

**Comment by rychlis** [comment]
We have just released Sketch Extension 3.0.0-prerelease4 with Sketch 3.5 support. Please download the latest version from https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md#installation--update

We're working on updating the older Sketch Plugin 2 however there has been some further issues with Sketch 3.5 unfortunately. You should be able to use the beta version fine though

**Comment by sawao** [comment]
Thank you so much :pray:
